frontend/src/modules/member/components/member-merge-suggestions-dialog.vue

```diff
@@ -5,9 +5,22 @@
     title="Merge suggestions"
     size="2extra-large"
     custom-class="mt-6 mb-6"
+    :show-header="false"
   >
```

Comment:
I think Nuno improved the UI of the dialog a bit, on the profiles boxes. This is his suggestion:
Could you make sure it's aligned with this?
- Dialog is centered on the page (excluding menu)
- margin-top is also fixed
- No border around profile boxes

<img width="817" alt="Screenshot 2024-04-16 at 16 39 02" src="https://github.com/CrowdDotDev/crowd.dev/assets/20134207/a395cdc2-3e06-4f57-a8af-2e7f12526cf9">